robinklaassen/aas2
# Added camp prices per year

What has been done
- Added pivot functionality in graphs
- Added graph for camp prices per year
  - Grouped per "type" (first x letters of the camp code)
  - Only currently for type 'camp'
  - Per year

How to test
- Go to the graphs page
- Go to "Kamp Prijzen"
- Check graph
